go/vt/vtorc/logic/vtorc.go

```diff
@@ -326,12 +326,6 @@ func refreshAllInformation(ctx context.Context) error {
 		return RefreshAllKeyspacesAndShards(ctx)
 	})
 
-	// Refresh shards to watch.
-	eg.Go(func() error {
-		updateShardsToWatch()
-		return nil
-	})
-
```

Comment:
We don't need this anymore, because we don't store the explicit shard names for the keyspaces that we want to watch in the entirety. Instead we just store a complete key range and that doesn't change during the course of running of a VTOrc instance.